patch/kconfig-inclusions

```diff
@@ -88,12 +87,57 @@ CONFIG_MLXREG_IO=m
 CONFIG_MAX1363=m
 CONFIG_THERMAL_NETLINK=y
 CONFIG_THERMAL_STATISTICS=n
-
 CONFIG_NVSW_SN2201=m
 CONFIG_TI_ADS1015=m
 CONFIG_SENSORS_EMC2305=m
 CONFIG_SENSORS_JC42=m
 CONFIG_SENSORS_POWR1220=m
+CONFIG_NET_VENDOR_MELLANOX=y
```

Comment:
Many of these configs appear to be already enabled or compiled as a module on the current 5.10.140 kernel. Do they really need to be specified here?

Reply:
Hi @saiarcot895, as @vivekrnv  communicated with you offline, we are using an automated tool to integrate the hw-mgmt and the tool doesn’t identify the kconfigs that are already present in the kernel. Since the redundant kconfigs are not going to have a functional impact, we want to merge this PR ASAP and we will address this in the future.